pkg/operator/starter.go

```diff
@@ -260,6 +260,7 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 		"OpenshiftAuthenticationStaticResources",
 		assets.Asset,
 		[]string{
+			"oauth-openshift/audit-policy.yaml",
```

Comment:
I am not sure if I am missing something here, but it seems like the configmap is not there, when I push the CAO image into the cluster. Am I missing something?

@emilym1 mentioned that there are no static configmaps anymore?